tests/test_host_check.py

```diff
@@ -750,77 +822,77 @@ class TestCPUCores(_CheckTestBase):
-        success, output = self.perform_check(capsys, cmd_effects="CPU(s): 1 ")
-        assert output == textwrap.dedent(
+        result, output = self.perform_check(capsys, cmd_effects="CPU(s): 1 ")
+        assert output == pretty_print(
             f"""\
-            FAIL -- CPU cores
-                    The number of available CPU cores is 1,
-                    but at least 2 CPU cores are required.
+             FAIL -- CPU cores
+                     The number of available CPU cores is 1,
+                     but at least 2 CPU cores are required.
             """
         )
-        assert not success
+        assert not result.is_success()
 
     def test_parse_error(self, capsys):
         """Test error when parsing output from 'lscpu'."""
-        success, output = self.perform_check(
+        result, output = self.perform_check(
             capsys, cmd_effects="bananas taste awesome"
         )
-        assert output == textwrap.dedent(
+        assert output == pretty_print(
             f"""\
-            WARN -- CPU cores
-                    Unable to parse the output from 'lscpu' -
-                    unable to check the number of available CPU cores.
-                    At least 2 CPU cores are required.
+            ERROR -- CPU cores
+                     Unable to parse the output from 'lscpu' -
+                     unable to check the number of available CPU cores.
+                     At least 2 CPU cores are required.
             """
         )
-        assert not success
+        assert not result.is_success()
 
     def test_subproc_error(self, capsys):
         """Test subprocess error being raised."""
-        success, output = self.perform_check(
+        result, output = self.perform_check(
             capsys, cmd_effects=subprocess.SubprocessError
         )
-        assert output == textwrap.dedent(
+        assert output == pretty_print(
             f"""\
-            WARN -- CPU cores
-                    Error running 'lscpu' to check the number of available CPU cores.
-                    At least 2 CPU cores are required.
+            ERROR -- CPU cores
+                     Error running 'lscpu' to check the number of available CPU cores.
+                     At least 2 CPU cores are required.
             """
         )
-        assert not success
+        assert not result.is_success()
 
     def test_unexpected_error(self, capsys):
         """Test unexpected error being raised."""
-        success, output = self.perform_check(
+        result, output = self.perform_check(
             capsys, cmd_effects=Exception("test exception")
         )
-        assert output == textwrap.dedent(
+        assert output == pretty_print(
             f"""\
-            FAIL -- CPU cores
-                    Unexpected error: test exception
+             FAIL -- CPU cores
+                     Unexpected error: test exception
             """
         )
```

Comment:
Ah this is a bit of an annoyance. I'd suggest doing `dedent(output) == dedent(...)` instead of this though (`pretty_print()` isn't really a descriptive name, it's not printing anything, and the purpose isn't making it pretty).